src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $trans-dur;
```

Comment:
Don't set `transition` for all properties, in your case you need only transform and color.